openapi/social/social.yaml

```diff
@@ -102,6 +102,9 @@ paths:
       description: Please contact Sales to learn more about the scheduling message API
       tags:
         - Messages
+      responses:
+        '200':
+          description: OK
```

Comment:
Adding this fixes the error:

```Info                                         server/main.go:141  Creating GRPC server...
Critical                                     server/main.go:75   Server startup failed at step named 'initializing /platform/ routes' with exit code 7. Error: invalid OAS file: openapi/platform.yaml
error unmarshaling JSON: Error while unmarshalling property 'components' (*openapi3.Components): Error while unmarshalling property 'schemas' (*map[string]*openapi3.SchemaRef): Error while unmarshalling property 'properties' (*map[string]*openapi3.SchemaRef): Error while unmarshalling property 'properties' (*map[string]*openapi3.SchemaRef): Error while unmarshalling property 'type' (*string): json: cannot unmarshal array into Go value of type string
exit status 7
```

Reply:
We don't normally vendor the master version of this file into the server. It's good to know that the fake path we created for documentation purposes causes issues on the server. 
This is an ok change to make.